modules/apps/portal-search/portal-search-web/src/main/resources/META-INF/resources/js/components/search_bar_configuration_suggestions/contributor_input_set_item/inputs/ContentTypeInput.js

```diff
@@ -119,100 +49,80 @@ export default function ContentTypeInput({onBlur, onChange, touched, value}) {
 		event.preventDefault();
 	};
 
-	const _handleClickRemove = () => {
-		onChange('');
-
-		onBlur();
-	};
-
-	const _handleClose = () => {
-		onOpenChange(false);
-	};
-
-	const _handleOpen = () => {
-		onOpenChange(true);
-	};
-
 	return (
-		<>
-			{open && (
-				<ContentTypeModal
-					initialSelectedType={value}
-					observer={observer}
-					onBlur={onBlur}
-					onChange={onChange}
-					onClose={_handleClose}
-				/>
-			)}
-
-			<ClayInput.GroupItem
-				className={getCN({
-					'has-error': !value && touched,
-				})}
-			>
-				<label>
-					{Liferay.Language.get('content-type')}
-
-					<span className="reference-mark">
-						<ClayIcon symbol="asterisk" />
+		<ClayInput.GroupItem
+			className={getCN({
+				'has-error': !value && touched,
+			})}
+		>
+			<label>
+				{Liferay.Language.get('content-type')}
+
+				<span className="reference-mark">
+					<ClayIcon symbol="asterisk" />
+				</span>
+
+				<ClayTooltipProvider>
+					<span
+						className="c-ml-2"
+						data-tooltip-align="top"
+						title={Liferay.Language.get('content-type-help')}
+					>
+						<ClayIcon symbol="question-circle-full" />
 					</span>
+				</ClayTooltipProvider>
+			</label>
 
-					<ClayTooltipProvider>
-						<span
-							className="c-ml-2"
-							data-tooltip-align="top"
-							title={Liferay.Language.get('content-type-help')}
-						>
-							<ClayIcon symbol="question-circle-full" />
-						</span>
-					</ClayTooltipProvider>
-				</label>
-
+			<ClayInput.Group>
 				<ClayInput.Group>
-					<ClayInput.Group>
-						<ClayInput.GroupItem prepend>
+					<ClayDropDown
```

Comment:
Instead of using `ClayDropDown` we should now be using the `Picker` component for better accessibility. It's already being used in portal as well and the PEDs team has been migrating some of the other dropdowns too. https://clayui.com/docs/components/picker.html

As an added note: For the `Picker` component, be aware of the "controlled" vs "uncontrolled" props. For example, if you use the `onSelectionChange` this makes it "controlled" meaning you have to control the `selectedKey` prop as well (rather than relying on out of the box functionality inside of the Picker component itself).

Reply:
I hadn't heard of Picker, it's very helpful! I was adding it in but wanted to ask about two things:

- The inputs would change to a lighter color, is that ok for the design? "Activity" is a similar dropdown, and it currently is grey. Should we be changing them all for consistency?
<img width="781" alt="Screenshot 2023-11-14 at 12 10 03 PM" src="https://github.com/liferay-search/liferay-portal/assets/14063502/a84c89b1-da47-4338-986e-d17ad0335131">

- Is there a chance Picker can support being "required"? The content type is a required field but it looks like it does not work to pass in `onBlur` or `required` into Picker.

Reply:
We should change wherever is necessary. We shouldn't be using dropdowns as a select picker anymore.

You can probably check if a value is selected by checking if `selectedKey` is not empty.

Reply:
This is the epic where they're trying to migrate everything where possible. https://liferay.atlassian.net/browse/LPS-194008

I suppose if there's no supported styling for required yet, we can keep the dropdown implementation for now, but maybe you can reach out to t-dxp-platform-experience (PEDs) team for advice how to add required/error styling.

Reply:
Pushed up the changes to change Content Type and Time Range to Clay Picker, thanks for the advice on using a custom trigger to handle required input!

I can convert the "Activity" dropdown later, after https://liferay.atlassian.net/browse/LPS-199988.